frontend/app/routes/_gcweb-app._index.tsx

```diff
@@ -8,7 +8,7 @@ import { getTypedI18nNamespaces } from '~/utils/locale-utils';
 
 import { LandingPageLink } from '~/components/landing-page-link';
 
-const i18nNamespaces = getTypedI18nNamespaces('common', 'gcweb');
+const i18nNamespaces = getTypedI18nNamespaces('common', 'gcweb', 'landingpage');
```

Comment:
I don't see any `gcweb` i18n keys being used in this file. If that's the case, `gcweb` can be removed here.